src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+
+You can use the `cockroach encode-uri` command to generate a connection string containing a cluster's certificate.
+
+For example, in this tutorial, you'll need a connection string for the source cluster when you start the replication stream from the destination cluster.
+
+1. On the source cluster in a new terminal window, generate a connection string, by passing the replication user, node IP, and port, along with the directory to the source cluster's CA certificate:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach encode-uri {replication user}:{password}@{node IP}:26257 --ca-cert {path to CA certificate} --inline
+    ~~~
+
+    The connection string output contains the source cluster's certificate:
+
+    ~~~
+    {replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A
+    ~~~
+
+1. In the SQL shell on the destination cluster, create an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) using the source cluster's connection string. Prefix the `postgresql://` scheme to the connection string and replace `{source}` with your external connection name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION {source} AS 'postgresql://{replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A`;
+    ~~~
+
+## Step 3. Start the LDR stream
+
+In this step, you'll start the LDR stream. You can replicate one or multiple tables in a single LDR stream. You cannot replicate system tables in LDR streams, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
+
+_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
+
+- `immediate` (default): Inserts data at the [KV layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}), which means that the stream does not encounter issues with updates and deletes. `immediate` mode does not insert at the [SQL layer]({% link {{ page.version.version }}/architecture/sql-layer.md %}), which means that the replication will not follow SQL constraints. As a result, you can only use `immediate` mode if you do not have [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) constraints on the replicating table. `immediate` mode provides faster replication compared to `validated` mode.  
```

Comment:
@msbutler do you have a suggestion on how this might be better stated? "stream does not encounter issues with updates and deletes." I dont want to give the vibe that the stream just never works with updates and deletes, but we have that weird update/delete conflict in SQL mode

Reply:
It'd be great to get these descriptions in perfect shape so we can add to an include file — we'll need these descriptions on other LDR pages.